app/models/location.rb

```diff
@@ -13,6 +13,16 @@ class Location < ApplicationRecord
     where("longitude IS NOT NULL").
     where("name <> ''")
   }
+  scope :newest, -> { order(created_at: :desc) }
+  scope :in_bounds, ->(sw_lat, sw_lng, ne_lat, ne_lng) do
+    where (
+      <<-EOS.squish
+        ((:sw_lat < :ne_lat AND latitude BETWEEN :sw_lat AND :ne_lat) OR (:ne_lat < :sw_lat AND latitude BETWEEN :ne_lat AND :sw_lat))
```

Comment:
Nice that you are taking care of above/below the equator... why no tests?  This should have some unit tests!

Reply:
Unit tests help explain why you need the `:sw_lat < :ne_lat`...